components/header.tsx

```diff
@@ -0,0 +1,52 @@
+'use client'
+import React, { useState } from 'react';
+import Link from 'next/link';
+import { AiOutlineMenu, AiOutlineSearch } from 'react-icons/ai';
+
+const Header = () => {
+  const [showMenu, setShowMenu] = useState(false);
+
+  const toggleMenu = () => {
+    setShowMenu((prev) => !prev);
+  };
+
+  return (
+    <nav className='bg-[#131C27] text-white flex flex-col md:flex-row justify-between px-4 py-2 md:py-3 w-full fixed z-20'>
+      <div className='flex justify-between'>
+        <h1 className='font-poller tracking-wider font-semibold text-3xl text-[#797DA5] md:px-4 md:text-4xl'>Nomads.</h1>
+        <div className='md:hidden text-3xl'>
+          <button onClick={toggleMenu} className='text-white'>
+            <AiOutlineMenu />
+          </button>
+        </div>
+      </div>
+      <ul
+        className={`${showMenu ? 'flex' : 'hidden'
+          } md:flex flex-col md:flex-row tracking-wider md:gap-10 md:text-xl font-abhaya text-xl md:mt-2`}
+      >
+        <li>
+          <Link href={'#'}>Destinations</Link>
+        </li>
+        <li>
+          <Link href={'#'}>Things To Do</Link>
+        </li>
+        <li>
+          <Link href={'#'}>Tours</Link>
+        </li>
+        <li>
+          <Link href={'#'}>Services</Link>
+        </li>
+      </ul>
+      <div className='flex mt-6 md:mt-0 rounded-full px-1 mx-6 justify-center items-center w-4/3 md:w-1/6 bg-slate-50'>
+        <AiOutlineSearch className='text-gray-700 text-3xl' />
+        <input
+          type='text'
+          placeholder='Search...'
+          className='font-aloe text-xl rounded-full py-1 bg-slate-50 px-1 outline-none border-0 w-full text-gray-900'
+        />
+      </div>
+    </nav>
```

Comment:
- [ ] @NurkaAmre You did an excellent job implementing the header part based on the Figma design, however, it would be nice if you wrap your navigation inside the `<header>...</header>` so that every page has one Semantic header tag available.